crypto/time.c

```diff
@@ -40,10 +40,9 @@ OSSL_TIME ossl_time_now(void)
         return ossl_time_zero();
     }
     if (t.tv_sec <= 0)
-        r.t = t.tv_usec <= 0 ? 0 : t.tv_usec * (OSSL_TIME_SECOND / 1000000);
+        r.t = t.tv_usec <= 0 ? 0 : t.tv_usec * OSSL_TIME_US;
     else
-        r.t = ((uint64_t)t.tv_sec * 1000000 + t.tv_usec)
-              * (OSSL_TIME_SECOND / 1000000);
-#endif
+        r.t = ((uint64_t)t.tv_sec * 1000000 + t.tv_usec) * OSSL_TIME_US;
```

Comment:
Is there something that could replace the `1000000`?

Reply:
The replacement is: `(OSSL_TIME_SECOND / OSSL_TIME_US)` which is hideous :grimacing:  I'm find with this constant -- it's converting _s_ to _μs_ and OSSL_TIME isn't involved.

Reply:
I will update the μs to time conversions to use the helper.